postal/pylangclassifier.c

```diff
@@ -0,0 +1,163 @@
+#include <Python.h>
+#include <libpostal/libpostal.h>
+#include "pyutils.h"
+
+#if PY_MAJOR_VERSION >= 3
+#define IS_PY3K
+#endif
+
+struct module_state {
+    PyObject *error;
+};
+
+
+typedef struct language_classifier_response {
+    Py_ssize_t num_languages;
+    char **languages;
+    double *probs;
+} language_classifier_response_t;
+
+#ifdef IS_PY3K
+    #define GETSTATE(m) ((struct module_state*)PyModule_GetState(m))
+#else
+    #define GETSTATE(m) (&_state)
+    static struct module_state _state;
+#endif
+
+
+static PyObject *py_classify_lang_address(PyObject *self, PyObject *args, PyObject *keywords) {
+    PyObject *arg_input;
+
+    PyObject *result = NULL;
+
+    if (!PyArg_ParseTuple(args, "O:pylangclassifier", &arg_input)) {
+        return 0;
+    }
+
+    char *input = PyObject_to_string(arg_input);
+
+    if (input == NULL) {
+        return NULL;
+    }
+
+    libpostal_language_classifier_response_t *response = libpostal_classify_language(input);
```

Comment:
`language_classifier_response_t` is defined above, but here we use `libpostal_language_classifier_response_t`.